src/components/upload-promotion/UploadPromotionForm.tsx

```diff
@@ -46,10 +46,16 @@ export default function UploadPromotionForm(): ReactElement {
   const [form] = Form.useForm();
 
   const history = useHistory();
+  const authUser = useAuthUser();
 
   const [datesEffective, setDatesEffective] = useState<PromotionDates>({ start: '', end: '' });
   const [times, setTimes] = useState<PromotionTimes>({});
 
+  // TODO: https://promopal.atlassian.net/browse/PP-80
+  if (!authUser) {
+    return <p>Error: No user is logged in.</p>;
+  }
```

Comment:
We're guaranteed that auth User is defined right, since the only way to get into this route is for auth user to be defined, otherwise it takes us to login form right

Reply:
Right, these if (!authUser) checks are only to see if a user illegally accesses the page